api/services/GithubBuildHelper.js

```diff
@@ -131,4 +131,19 @@ const reportBuildStatus = (build) => {
     });
 };
 
-module.exports = { reportBuildStatus };
+const fetchContent = async (build, path = null) => {
```

Comment:
it looks like `path` is required so we shouldn't have a default argument

Reply:
nice catch 👍